src/js/components/MultiSelect/MultiSelect.js

```diff
@@ -0,0 +1,504 @@
+    const inputValue = useMemo(() => {
+      if (!selectValue) {
+        if (optionIndexesInValue.length === 0) return '';
+        if (optionIndexesInValue.length === 1)
+          return applyKey(allOptions[optionIndexesInValue[0]], labelKey);
+        if (messages) return format({ id: 'select.multiple', messages });
+        return `${optionIndexesInValue.length} selected`;
+      }
+      return undefined;
+    }, [
+      labelKey,
+      messages,
+      format,
+      optionIndexesInValue,
+      allOptions,
+      selectValue,
+    ]);
+
+    const iconColor = getIconColor(theme);
+
+    const dropContent = (
+      <MultiSelectContainer
+        allOptions={allOptions}
+        disabled={disabled}
+        disabledKey={disabledKey}
+        dropHeight={dropHeight}
+        emptySearchMessage={emptySearchMessage}
+        help={help}
+        id={id}
+        labelKey={labelKey}
+        limit={limit}
+        onChange={onSelectChange}
+        onClose={onRequestClose}
+        onKeyDown={onKeyDown}
+        onMore={onMore}
+        onSearch={onSearch}
+        options={optionsProp}
+        optionIndexesInValue={optionIndexesInValue}
+        replace={replace}
+        searchPlaceholder={searchPlaceholder}
+        search={search}
+        setSearch={setSearch}
+        usingKeyboard={usingKeyboard}
+        value={value}
+        valueKey={valueKey}
+        showSelectedInline={showSelectedInline}
+      >
+        {children}
+      </MultiSelectContainer>
+    );
+
+    const dropButtonProps = {
+      ref: dropButtonRef,
+      a11yTitle: `${ariaLabel || a11yTitle || placeholder || 'Open Drop'}. ${
+        value.length
+      } selected.`,
+      'aria-expanded': Boolean(open),
+      'aria-haspopup': 'listbox',
+      id,
+      disabled: disabled === true || undefined,
+      open,
+      focusIndicator,
+      onFocus,
+      onBlur,
+      gridArea,
+      margin,
+      onOpen: onRequestOpen,
+      onClose: onRequestClose,
+      onClick,
+      callerPlain: plain,
+      plain, // Button should be plain
+      dropProps,
+      dropContent,
+      theme,
+    };
+
+    const displaySelectIcon = SelectIcon && (
+      <Box
+        alignSelf="center"
+        margin={theme.select.icons.margin}
+        style={{ minWidth: 'auto' }}
+      >
+        {isValidElement(SelectIcon) ? (
+          SelectIcon
+        ) : (
+          <SelectIcon color={iconColor} size={size} />
+        )}
+      </Box>
+    );
+
+    return (
+      <Keyboard onDown={onRequestOpen} onUp={onRequestOpen}>
+        {showSelectedInline ? (
+          <StyledSelectBox
+            disabled={disabled === true || undefined}
+            alignSelf={alignSelf}
+            direction="row"
+            alignContent="start"
+            background={theme.select.background}
+            ref={selectBoxRef}
+            flex={false}
+            callerPlain={plain}
+            width={width}
+          >
+            <Box direction="column" width="100%">
```

Comment:
the column is the default direction can we just strip that out?

Reply:
Yes, fixed